src/plugins/ima/index.js

```diff
@@ -1035,6 +1023,25 @@
       }
     }.bind(this);
 
+    var setAdMuted = function(mute) {
+      if (mute) {
+        addClass_(this.muteDiv, 'ima-muted');
+        removeClass_(this.muteDiv, 'ima-non-muted');
+        this.adsManager.setVolume(0);
+        // Bubble down to content player
+        this.player.muted(true);
+        this.adMuted = true;
+        this.sliderLevelDiv.style.width = "0%";
+      } else {
+        addClass_(this.muteDiv, 'ima-non-muted');
+        removeClass_(this.muteDiv, 'ima-muted');
+        this.adsManager.setVolume(1);
```

Comment:
Shouldn't you use the current volume of the player rather then set volume to 100%?